use-build-os-config.rst

```diff
@@ -0,0 +1,57 @@
+.. post:: August 7, 2023
+   :tags: builders, deprecation
+   :author: Manuel
+   :location: BCN
+   :category: Changelog
+
+
+Use ``build.os`` instead of ``build.image`` on your configuration file
+======================================================================
+
+We are announcing the deprecation of ``build.image`` config key in favor of ``build.os``.
+Read the Docs *will start requiring* a ``build.os`` config key for all projects in order to build documentation successfully.
+**We will start failing builds using "build.image" on their config file on October 16, 2023**.
+
+
+Deprecation timeline
+--------------------
+
+We understand this change will affect many of our users,
+so we have a timeline to communicate this deprecation to our users effectively.
+
+* **Monday, August 28, 2023**: Do the first brownout (temporarily enforce this deprecation) for 12 hours: 00:01 PST to 11:59 PST (noon)
+* **Monday, September 18, 2023**: Do a second brownout (temporarily enforce this deprecation) for 24 hours: 00:01 PST to 23:59 PST (midnight)
+* **Monday, October 2, 2023**: Do a third and final brownout (temporarily enforce this deprecation) for 48 hours: 00:01 PST to October 3, 2023 23:59 PST (midnight)
```

Comment:
Is it possible to sync up any of these brownouts with our existing brownouts for the config file deprecation? I feel like having a brownout almost every month for the next couple months is a lot. It would be good if we could just codify the brownout (eg. `if date(2023, 08, 28) < datetime.now() date(2023, 08, 29)` in the code vs. a feature flag)

Reply:
Cool. I've done this in [458913eab209217a17791b721e2eb5dfe4b33f1a](https://github.com/readthedocs/readthedocs.org/pull/10589/commits/458913eab209217a17791b721e2eb5dfe4b33f1a)